### ISSUE TITLE
Frequent stuttering in IDE and testing game in 2024.4

### ISSUE THREAD

**Issue by backYard321** [YoYoGames/GameMaker-Bugs#5763]
### Description
When using gamemaker and and testing my game using "run", gamemaker will frequently stutter, resulting in dropped keystrokes and laggy gameplay, despite the profiler not showing drops in performance. Exporting my game as a .exe and playing that does not show these issues.

### Which version of GameMaker are you reporting this issue for?
IDE v2024.4.0.137 Runtime v2024.4.0.168

### Which operating system(s) are you seeing the problem on?
Windows 10.0.19045.0

[3611fe56-69a1-4c82-904c-00d6a02dbc64](https://api.gamemaker.io/api/github/downloads/3611fe56-69a1-4c82-904c-00d6a02dbc64)

**Comment by stuckie** [comment]
You have various errors in your ui.log about a custom theme:

`[10:20:11:421(d2a7)] Failed to load theme at 'C:\ProgramData/GameMakerStudio2\Skins/Dracula\Themes\default_theme.json', with exception 'After parsing a value an unexpected character was encountered: l. Path 'ColourPalette.ins_table_stripe', line 17, position 16.'
[10:20:11:422(d2a7)] ####################################
[10:20:11:422(d2a7)] Break in LoadTheme (ThemeManager:102)
[10:20:11:422(d2a7)] ####################################
[10:20:12:134(d2a7)] File exception 'Invalid character after parsing property name. Expected ':' but got: [. Path 'ins_expanded_icon.height', line 879, position 26.'
[10:20:12:135(d2a7)] ####################################
[10:20:12:135(d2a7)] Break in StyleLoadError (StyleManager:273)
[10:20:12:136(d2a7)] Reason: StyleManager: Style error "C:\ProgramData\GameMakerStudio2\Skins\Dracula\Styles\styles_inspector.json": Invalid character after parsing property name. Expected ':' but got: [. Path 'ins_expanded_icon.height', line 879, position 26.
[10:20:12:136(d2a7)] ####################################`

I cannot reproduce any slow down here on the standard themes, could you verify that those work fine?

**Comment by backYard321** [comment]
> You have various errors in your ui.log about a custom theme:
> 
> `[10:20:11:421(d2a7)] Failed to load theme at 'C:\ProgramData/GameMakerStudio2\Skins/Dracula\Themes\default_theme.json', with exception 'After parsing a value an unexpected character was encountered: l. Path 'ColourPalette.ins_table_stripe', line 17, position 16.' [10:20:11:422(d2a7)] #################################### [10:20:11:422(d2a7)] Break in LoadTheme (ThemeManager:102) [10:20:11:422(d2a7)] #################################### [10:20:12:134(d2a7)] File exception 'Invalid character after parsing property name. Expected ':' but got: [. Path 'ins_expanded_icon.height', line 879, position 26.' [10:20:12:135(d2a7)] #################################### [10:20:12:135(d2a7)] Break in StyleLoadError (StyleManager:273) [10:20:12:136(d2a7)] Reason: StyleManager: Style error "C:\ProgramData\GameMakerStudio2\Skins\Dracula\Styles\styles_inspector.json": Invalid character after parsing property name. Expected ':' but got: [. Path 'ins_expanded_icon.height', line 879, position 26. [10:20:12:136(d2a7)] ####################################`
> 
> I cannot reproduce any slow down here on the standard themes, could you verify that those work fine?

Still getting the same issues using both the dark and light themes, unfortunately. 

I'm checking a couple things - it miiight have to do with having too many assets/workspaces open at the same time? When I closed everything and updated to 2024.4, everything seemed to run fine, but I experienced stuttering again after a while. Creating a new workspace + "close all but this" seemed to eliminate the stuttering when I ran the game immediately after that.

**Comment by backYard321** [comment]
I don't experience this issue in 2024.6, so I think the underlying problem is fixed. Thanks!

**Comment by stuckie** [comment]
Good to know!
Would have been nice to point the finger at a specific cause, but if things have resolved, that's enough for now.

Thanks.

**Comment by backYard321** [comment]
yeah unfortunately I couldn't really isolate the cause, and at a certain point i just didn't have any more time to troubleshoot!
